iocore/net/QUICNetVConnection.cc

```diff
@@ -204,28 +203,40 @@ QUICNetVConnection::free(EThread *t)
 {
   QUICConDebug("Free connection");
 
-  this->_ctable->erase(this->_original_quic_connection_id, this);
-  this->_ctable->erase(this->_quic_connection_id, this);
-  for (unsigned int i = 0; i < countof(this->_alt_quic_connection_ids); ++i) {
-    this->_ctable->erase(this->_alt_quic_connection_ids[i].id, this);
-  }
-
   /* TODO: Uncmment these blocks after refactoring read / write process
```

Comment:
We can uncomment these block because ref counting guarantees that `free()` won't be called before there are users. I just thought removing current READ_READY, WRITE_READY events would solve the issue.